### PR TITLE
Peer code review PR 👩🏻‍💻

### PR DESCRIPTION
**Pull request by becarolf**
hey there, thanks for reviewing my code! 😬

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,31 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.scss";
+import { useEffect, useState } from "react";
+import firebase from "./firebase";
+import { getDatabase, ref, onValue } from "firebase/database";
+import Header from "./Components/Header";
+import Quiz from "./Components/Quiz";
+import Footer from "./Components/Footer";
```
Comment (clairehousez) [inline src/App.js:7]: I like how you made separate components for the header and footer, that's smart!

File: src/App.js [hunk 2]
```diff
@@ -1,25 +1,31 @@
...
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div id="page-container">
+      <div id="content-wrap">
```
Comment (clairehousez) [inline src/App.js:22]: yay for wrappers! ✨

File: src/App.scss [hunk 3]
```diff
@@ -0,0 +1,153 @@
+// variables
```
Comment (clairehousez) [inline src/App.scss:1]: Great job using Sass!!! I got too stressed to figure it out, so good on you

File: src/App.scss [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+h2 {
+  font-size: 54px;
+  font-style: italic;
+  text-decoration: overline;
+  margin: 0;
+}
+
+.tv-header {
+  background-image: url(./assets/pattern_overlapping-hexagons_0.5_2_0-0_0_1__cdc6ac_ffffff.png);
+  //   background-repeat: no-repeat;
```
Comment (clairehousez) [inline src/App.scss]: Could maybe remove these if you didn't end up needing them

File: src/App.scss [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+}
+
+#page-container {
+  position: relative;
+  min-height: 100vh;
+}
+
+#content-wrap {
+  max-width: 1280px;
+  padding-bottom: 2.5rem; /* Footer height */
```
Comment (clairehousez) [inline src/App.scss]: Oh, this is smart. That may fix some of the footer issues I was having. Good job!!

File: src/App.scss [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+    width: 80%;
+    height: 50px;
+    margin: 10px;
+    border: none;
+    border-radius: 10px;
+    font-family: $quizFont;
+    font-size: 20px;
+    text-transform: uppercase;
+    &:hover,
+    &:focus {
```
Comment (clairehousez) [inline src/App.scss:113]: Good job making sure the button is accessible ✨

File: src/App.scss [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+
+.wrong-answer {
+  border: 5px solid red;
+}
+
+.next-button {
+  width: 200px;
+  height: 45px;
+  margin: 20px 0;
+  float: right;
```
Comment (clairehousez) [inline src/App.scss:133]: I've heard floats are not really used anymore, but if it works for you that's good!

File: src/App.scss [hunk 8]
```diff
@@ -0,0 +1,153 @@
...
+  background-color: #f0f0f3;
```
Comment (clairehousez) [inline src/App.scss:134]: Could save this colour into a variable as well

File: src/App.scss [hunk 9]
```diff
@@ -0,0 +1,153 @@
...
+      background-color: transparent;
+      max-width: 550px;
+      position: relative;
+      top: 6px;
+      left: 10px;
+    }
+  }
+}
+
+#page-container {
```
Comment (clairehousez) [inline src/App.scss]: They might deduct marks for using an ID tag # for styling, you can add a class name to the element, in addition to an ID

File: src/App.scss [hunk 10]
```diff
@@ -0,0 +1,153 @@
...
+    box-shadow: none;
+  }
+}
+
+.answer-container {
+  .youtube {
+    padding-top: 80px;
+  }
+}
+#footer {
```
Comment (clairehousez) [inline src/App.scss:156]: if it's just the footer, you don't need the ID tag #, I think you can just write "footer" without a class or ID

File: src/Components/Header.js [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+        <header className="tv-header">
+            <div className="header-container">
+                <div className="header-text">
+                <h1>Guess the Song</h1>
+                <h2>Taylor's Version</h2>
+            </div>
+            
+            <div className="image-container">
+                <img className="header-image" src=
+                {TaylorPic} alt="Taylor Swift wearing sunglasses and a red lipstick." />
```
Comment (clairehousez) [inline src/Components/Header.js]: Nice alt tag!

File: src/Components/Quiz.js [hunk 12]
```diff
@@ -0,0 +1,84 @@
...
+    }
+  };
+
+  useEffect(() => {
+    setNewQuestion(questions);
+  }, [questions]);
+
+  const handleAnswer = (e) => {
+    const userAnswer = e.target.innerText;
+    setAnswer(userAnswer.toUpperCase() === currentQuestion.answer.toUpperCase());
```
Comment (clairehousez) [inline src/Components/Quiz.js]: ooo smart thinking, with the .toUpperCase

File: src/Components/Quiz.js [hunk 13]
```diff
@@ -0,0 +1,84 @@
...
+  };
+
+  const handleNext = () => {
+    if (!questionsControl.length) {
```
Comment (clairehousez) [inline src/Components/Quiz.js:32]: Great error handling here!!

File: src/questions.json [hunk 14]
```diff
@@ -0,0 +1,321 @@
...
+
+    {
+        "id": 32,
+        "question": "And I've cleaned enough houses to know how to cover up a scene. Good thing Este's sister's gonna swear she was with me",
+        "option1": "No Body, No Crime",
+        "option2": "Delicate",
+        "option3": "'Evermore",
+        "answer": "No Body, No Crime",
+        "lyricsVideo": "https://www.youtube.com/embed/IEPomqor2A8"
+    }
```
Comment (clairehousez) [inline src/questions.json:320]: Awesome job!! I love that you embeded a video for each question, that's really cool!! You did a really great job on everything!!

**Review by clairehousez — APPROVED** [review]
Amazing job, Bea!!

I gave you some small styling notes, but otherwise I think you did a fantastic job!

- Claire & Bear

File: src/Components/Quiz.js (hunk 12 above)
Comment (becarolf) [reply src/Components/Quiz.js]: I changed to uppercase first on my scss file then my quiz broke. It took me a while to figure out what I did wrong 😮‍💨 God bless the console.log lol

**Pull request merged**